plugins/processors/noise/README.md

```diff
@@ -76,5 +98,10 @@ Result of noise added to the _cpu_ metric:
 
 ```diff
 - cpu map[cpu:cpu11 host:98d5b8dbad1c] map[usage_guest:0 usage_guest_nice:0 usage_idle:94.3999999994412 usage_iowait:0 usage_irq:0.1999999999998181 usage_nice:0 usage_softirq:0.20000000000209184 usage_steal:0 usage_system:1.2000000000080036 usage_user:4.000000000014552]
-+ cpu map[cpu:cpu11 host:98d5b8dbad1c] map[usage_guest:1.0078071583066057 usage_guest_nice:0.523063861602435 usage_idle:95.53920223476884 usage_iowait:0.5162661526251292 usage_irq:0.7138529816101375 usage_nice:0.6119678488887954 usage_softirq:0.5573585443688622 usage_steal:0.2006120911289802 usage_system:1.2954475820198437 usage_user:6.885664792615023]
++ cpu map[cpu:cpu11 host:98d5b8dbad1c] map[usage_guest:1.0078071583066057
+usage_guest_nice:0.523063861602435 usage_idle:95.53920223476884
+usage_iowait:0.5162661526251292 usage_irq:0.7138529816101375
+usage_nice:0.6119678488887954 usage_softirq:0.5573585443688622
+usage_steal:0.2006120911289802 usage_system:1.2954475820198437
+usage_user:6.885664792615023]
```

Comment:
If you view the rendered page this doesn't look right either. As this is code does it really need to be broken up?

Reply:
same as previous